plugin/src/main/java/com/denizenscript/denizen/objects/ColorTag.java

```diff
@@ -290,6 +293,9 @@ public static void register() {
         // Returns the Red, Green, and Blue values of this ColorTag as an integer number, equivalent to an integer reparse of <@link tag ColorTag.hex>.
         // Highest order bits are red, then green, then lowest is blue.
         // This is a rare special case encoding usually avoided by most systems, but may be necessary for some obscure tools.
+        // @example
+        // # Narrates "16711935".
+        // - narrate <color[fuchsia].rgb_integer>

```

Comment:
As said on the previous PR, if you don't know what a tag is for, don't jank an example of it.